src/librustc_typeck/check/method/probe.rs

```diff
@@ -1411,7 +1415,7 @@ impl<'a, 'gcx, 'tcx> ProbeContext<'a, 'gcx, 'tcx> {
         Some(Pick {
             item: probes[0].0.item.clone(),
             kind: TraitPick,
-            import_id: probes[0].0.import_id,
+            import_ids: probes[0].0.import_ids.clone(),
```

Comment:
Would it be hard to store a reference in `Pick` and avoid clones?

Reply:
I'm not sure. I used a smallvec to that it wouldn't be require heap alloc in the general case, and SmallVec doesn't implement Move like Option does.